rdrf/rdrf/utils.py

```diff
@@ -591,57 +591,106 @@ def __le__(self, other):
     def __eq__(self, other):
         return (self is other)
 
-def process_embedded_html(html, translate=True):
+def get_field_from_model(model_path):
+    # model_path looks like:  model/ConsentSection/23/information_text
+    # model must be in rdrf.models
+    from django.apps import apps
+    try:
+        parts = model_path.split("/")
+        model_name = parts[1]
+        pk = int(parts[2])
+        field = parts[3]
+        model_class = apps.get_model('rdrf', model_name)
+        model_instance = model_class.objects.get(pk=pk)
+        value =  getattr(model_instance, field)
+        return value
+    except Exception as ex:
+        logger.error("Error retrieving value from model_path %s: %s" % (model_path,
+                                                                        ex))
+        return
+        
+def get_registry_definition_value(field_path):
+    # find a value in a registry definition
+    # e.g.
+    # cde field:
+    # <reg code>/<form name>/<section code>/<cde code>/<cde field>
+    # section field:
+    # <reg code>/<form name>/<section code>/<section field>
+    # form field:
+    # <reg code>/<form name>/<form field>
+    # registry field
+    # <reg code>/<registry field>
+    # for consents , I have  introduced a more general notion:
+    # model/<ModelName>/<pk>/<fieldname> - delegated to get_field_from_model above
```

Comment:
I like the use of a sentinel at the beginning of the field_path to denote the type of field path as is the case with 'model'. I think an improvement would be to use that everywhere, consistently, not just with model as a special case.

For example:
 # cdefield/<reg code>/<form name>/<section code>/<cde code>/<cde field>
 # sectionfield/<reg code>/<form name>/<section code>/<section field>
 # formfield/<reg code>/<form name>/<form field>
 # registryfield/<reg code>/<registry field>

That would then enable the conditional logic to tighten as well (not needing to count args etc)